Cohort1/tbd/client/tbd/src/search/SearchPage.js

```diff
@@ -1,7 +1,14 @@
+import './index.css'; 
```

Comment:
These files are missing, which is why the netlify build is complaining. You probably don't need both the index.css and search.css file. So maybe combine them into one file and add it here.